content/en/blog/2023/otel-end-user-discussions-january-2023.md

```diff
@@ -0,0 +1,244 @@
+nuances. The Collector is not suited to address this.
+
+#### 5- Advanced Collector deployment and configuration
+
+**Q:** When should I be horizontally scaling my pod vs modifying config of an
+individual collector? When do I add more collectors or change collector config?
+
+**A:** There are a few things to consider when deploying and configuring
+Collectors.
+
+- If you only have stateless components in your Collector, you can scale (add
+  more replicas) based on metrics.
+- You may want to shard your pipelines based on the type of processing that
+  you’re doing. For example, creating one metrics pipeline, one logs pipeline,
+  and one traces pipeline, because the workload for each of these pipelines is
+  different.
+- You might want to split your Collectors based on the type of data being
+  processed. If there’s one namespace where there’s more data that comes in with
+  [personally identifiable information (PII)](https://www.investopedia.com/terms/p/personally-identifiable-information-pii.asp),
+  you might want to have a dedicated Collector for that namespace that uses the
+  [attributes processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor/README.md).
+
+### OpenTelemetry Adoption & Enablement
+
+**Q:** So you’ve decided to go with OpenTelemetry at your organization…now what?
+What’s the best way to promote OpenTelemetry adoption, and get developers
+excited about using OpenTelemetry, without overwhelming them?
+
+**A:** Some suggestions from the community:
+
+- Find folks who are willing to be OpenTelemetry champions
+- Pair developers new to OpenTelemetry with those who are more familiar with it
+- The real value of OpenTelemetry won’t be seen until you instrument a few
+  services, to see how things are stitched together.
+- Developers must be mentally ready to start instrumenting their code. Keep in
+  mind that it may mean going into existing code to instrument it.
+- A “big bang” approach may not be the best way to adopt OpenTelemetry, as it
+  may be too overwhelming for an organization. Start with a component or two.
+
+### OpenTelemetry Language API & SDKs
+
+#### 1- New language instrumentation
+
+**Q:** How do you find information on OTel implementations for different
+languages, for example, [Dart](https://dart.dev) and [Lua](https://www.lua.org)?
+
+**A:** [CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf) is
+always a good place to start your search. There are language-specific channels,
+which follow the naming convention **otel-&lt;language_name>**. If you don’t
+find a channel for your language, feel free to start a discussion on the
+[OpenTelemetry CNCF Slack channel](https://cloud-native.slack.com/archives/CJFCJHG4Q),
+or on [GitHub](https://github.com/open-telemetry/community), like with
+[this issue for OTel for Perl](https://github.com/open-telemetry/community/issues/828).
```

Comment:
Please reference this page: https://opentelemetry.io/docs/instrumentation/other/